tests/conftest.py

```diff
@@ -18,11 +18,35 @@
 def pytest_addoption(parser):
     """
     Additional command-line arguments passed to pytest.
-        --cpu: use CPU during testing (DEFAULT: GPU)
     """
     parser.addoption(
         "--cpu", action="store_true", help="pass that argument to use CPU during testing (DEFAULT: False = GPU)"
     )
+    parser.addoption("--flaky", action="store_true", default=False, help="Run flaky tests")
```

Comment:
What the benefit of using this vs `pytest -m "flaky"` or `pytest -m "not flaky"`?

Reply:
having said that: Cool and useful learning about `parser.addoption`! Will probably steal some of this stuff for Mcore!

Reply:
Nothing really, more of a convenience. One gap to this flag is you'd have to re-invent the more advanced conjunctive/disjunctive logic if someone wanted `pytest -m 'not flaky and (optimizer or checkpoint)'`.

Reply:
i've removed the flag